### ISSUE TITLE
 Bad  Request on clicking the photo link.

### ISSUE THREAD

**Issue by joshiamita1** [mozilla/hubs#3223]
**Description**
Bad  Request on clicking the photo link after entering the room.

**To Reproduce**
1. Go to 'Enter Room'
2. Click on ' Accept ->Enter on Screen->Enter Now -> click on Camera icon  ....'
3. click on '' took a photo"
4. See error " bad request"

**Expected behavior**
A clear photo of the avatar should be visible after clicking  on the photo link

**Screenshots**

![bad request](https://user-images.githubusercontent.com/33997040/97138952-69e78c80-177f-11eb-8c39-da8795b50b8e.jpg)

![on_click](https://user-images.githubusercontent.com/33997040/97138936-5a684380-177f-11eb-81cd-1e8dfebcdb9b.jpg)

**Hardware** 
 - Device: [Desktop]
 - OS: [e.g. Windows]
 - Browser [chorme- Version 86.0.4240.111 (Official Build) (64-bit)]

**Comment by misslivirose** [comment]
Do you have any popup blockers or plugins on your browser? I was not able to reproduce this.

**Comment by Softvision-GeluHaiduc** [comment]
I managed to reproduce the issue on `dev.reticulum.io` using chrome on windows.
After I took the picture and clicked the link the same “Bad request.” message appeared and I saw the URL, it was `https://dev.reticulum.io/https://hubs-upload-cdn.com/files/411c051f-0dee-415e-9ef2-10a350c87c56.html?token=595244f103b21fd7b182878ac8c4a936` with `https://dev.reticulum.io` being added to the front of the URL, if this part is removed the photo is loaded and can be seen, as to why this happens I don’t know. 
Below is a screen recording.
![photo_link](https://user-images.githubusercontent.com/51400741/99252216-4f04b700-2817-11eb-8242-a47383035423.gif)
